app/app/vhosts/edit/route.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default Ember.Route.extend({
+  title: function(){
+    let app = this.modelFor('app');
+
+    return `Edit ${this.currentModel.get('virtualDomain')} - ${app.get('handle')}`;
```

Comment:
Should be able to:

```
`Edit ${this.currentModel.get('virtualDomain')}`
```

And the title logic in:

https://github.com/bantic/diesel.aptible.com/blob/app-domain-edit/app/app/route.js#L4

Will take care of the app handle.